packages/core/src/components/Review/Review.example.jsx

```diff
@@ -2,33 +2,46 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import Review from './Review';
 
+const editButtonHref = 'javascript:void(0);';
```

Comment:
I added `editButtonHref` because `<a href="javascript:void(0);">` would fail eslint, because it requires a valid, navigable address. Open to suggestions on how to better solve this!

Reply:
I wonder why eslint didn't pick up on this before. Does it only run on staged files?

Reply:
My guess is because all the other javascript voids were saved to the `editHref` prop, while my example saved it directly to `href`